### PR TITLE
Add 100% button when sending or debonding ROSE 

### PR DESCRIPTION
**Pull request by buberdds**
Draft PR that targets https://github.com/oasisprotocol/oasis-wallet-web/issues/492

Screenshot 2022-01-03 at 10 26 43

Screenshot 2022-01-03 at 10 18 56
(disabled due to 0 tokens on this account)

### CODE REVIEW

File: src/app/components/AmountTextInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}: Props) {
+  return (
+    <>
+      <FormField
+        contentProps={inline ? contentProps : {}}
+        htmlFor="amount-id"
+        label={label}
+        name="amount"
+        validate={{
+          regexp: /^\d*((\.|,)\d{0,9})?$/,
```
Comment (buberdds) [inline src/app/components/AmountTextInput/index.tsx:41]: How many decimal places are allowed?  for example `AmountFormatter` defaults to 15, but it's not aligned with `10 ** 9`

File: src/app/pages/AccountPage/Features/SendTransaction/index.tsx [hunk 2]
```diff
@@ -42,7 +60,12 @@ export function SendTransaction() {
 
   return (
     <Box border={{ color: 'background-front-border', size: '1px' }} round="5px" background="background-front">
-      <Form>
+      <Form<SendTransactionFormState>
+        validate="submit"
```
Comment (buberdds) [inline src/app/pages/AccountPage/Features/SendTransaction/index.tsx:64]: I went with full `Form` API.  Props like `validate` and `onSubmit` look good here. I am not sure about typed form, `onChange` and `value`.  Should I  discard these and  restore

```
  const [recipient, setRecipient] = useState('')
  const [amount, setAmount] = useState('')
  ```
pattern ?

File: src/app/components/AmountTextInput/index.tsx (hunk 1 above)
Comment (pro-wh) [reply src/app/components/AmountTextInput/index.tsx:44]: should be 9, dunno why AmountFormatter mentions 15

Comment (buberdds) [reply src/app/components/AmountTextInput/index.tsx:44]: Ok, thanks. I will reopen PR once it's ready.

**Pull request closed without merging**